mappers/mqtt-mapper/resource/configmap.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
```

Comment:
The built-in mapper also needs to provide sample files of device model and device instance, such as https://github.com/kubeedge/mappers-go/blob/main/mappers/kubeedge-v1.17.0/onvif-mapper/resource/onvifdevice-instance.yaml   
https://github.com/kubeedge/mappers-go/blob/main/mappers/kubeedge-v1.17.0/onvif-mapper/resource/onvifdevice-model.yaml

Reply:
Thank you for your suggestion! I have added the model and instance files to the appropriate locations as requested.